data/jibauni/jibauni 0 kanlanmo transport.txt

```diff
@@ -0,0 +1,628 @@
+# Copyright (c) 2024 by Loymdayddaud
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Jibauni: Kanlanmo 0"
+	name "Take Kanlanmo to <planet>"
+	description "Kanlanmo, who is a Jibauni in the shape of a Tyrannosaurus Rex, has asked to go to <planet> on your ship. He will explain more once you get there."
+	landing
+	source "Baitoon"
+	destination "Taltahard"
+	on offer
+		conversation
+			`The approach to your landing location takes the ship uncomfortably close to huge trapezoidal skyscrapers, finally landing in the very midst of the city. The alien metropolis is filled with dinosaurs in the shape of the mythical Tyrannosaurus Rex, although here smaller then in old Earth fables. Soon after you land, one of these dinosaurs, incongruently wearing a dark green cloak, approaches your ship.`
```

Comment:
I don't think it's accurate to describe a T-Rex as 'mythical' or from 'fables'. That's how you'd describe a Troll. T-Rex was never in any myths, and would only be a character in modern 'fables'.